src/css/embed.scss

```diff
@@ -197,25 +212,12 @@ $brexit-control-button-size: 28px;
         }
     }
 }
-.brexit__nav-control--text {
-    @include fs-textSans(3);
-    font-weight: 900;
-    .brexit__nav-control__label {
-        padding: 0 $gutter;
-        width: auto;
-    }
-}
-
 .brexit__nav-control__label {
+    @include button($brexit-control-button-size);
```

Comment:
as a separate PR we can look at refactoring to use this `button()` mixin more widely